src/o365/spo/commands/page/page-get.ts

```diff
@@ -65,18 +65,16 @@ class SpoPageGetCommand extends SpoCommand {
           });
         });
 
-        const page: any = {
-          commentsDisabled: res.ListItemAllFields.CommentsDisabled,
-          numSections: clientSidePage.sections.length,
-          numControls: numControls,
-          title: res.ListItemAllFields.Title
-        };
-
         if (res.ListItemAllFields.PageLayoutType) {
-          page.layoutType = res.ListItemAllFields.PageLayoutType;
+          res.layoutType = res.ListItemAllFields.PageLayoutType;
         }
 
-        cmd.log(page);
+        res.commentsDisabled = res.ListItemAllFields.CommentsDisabled;
```

Comment:
The idea was to only output additional properties in JSON output mode not the default text mode. See the spec for #1729 for more information